QualityControl/lib/api.js

```diff
@@ -58,7 +59,12 @@ export const setup = (http, ws) => {
   );
 
   http.get('/status/gui', statusController.getQCGStatus.bind(statusController), { public: true });
-  http.get('/status/framework', statusController.getFrameworkInfo.bind(statusController), { public: true });
+  http.get(
+    '/status/:component',
+    statusComponentMiddleware,
+    statusController.getFrameworkInfo.bind(statusController),
```

Comment:
I believe the method name here has two issues:
- name is not representative, you are retrieving the status of a component which is defined in the URL
- for controllers, let's try to keep the convention of also ending in `handler` as name